### PR TITLE
LOTS!

### PR DESCRIPTION
**Pull request by Kyleeld**
AnimationID additions
ItemMapping fix

new plugin type: Skilling

Menu Entry Swapper grouping.

Menu Entry Additions:
claim dynamite
occult alter swap
obelisk swap
tears of guthix swap
buy plank swap
stun hoop snakes swap
chinchompa metamorphosis swap
escort swap
wilderness lever swap
search swap
hardwoodgrove swap

theiving fail/successfull /hr overlay

### CODE REVIEW

File: runelite-api/src/main/java/net/runelite/api/AnimationID.java [hunk 1]
```diff
@@ -155,7 +155,10 @@
 	public static final int SAND_COLLECTION = 895;
 	public static final int PISCARILIUS_CRANE_REPAIR = 7199;
 	public static final int HOME_MAKE_TABLET = 4067;
-
+	public static final int THIEVING_STALL = 832;
+	public static final int PICKPOCKET_SUCCESS = 881;
+	public static final int PICKPOCKET_FAIL = 424;
```
Comment (ThatGamerBlue) [inline runelite-api/src/main/java/net/runelite/api/AnimationID.java]: PICKPOCKET_FAIL is the same as BLOCK_UNARMED

File: runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java [hunk 2]
```diff
@@ -505,8 +505,8 @@
 	ITEM_ANGUISH_ORNAMENT_KIT(ANGUISH_ORNAMENT_KIT, NECKLACE_OF_ANGUISH_OR),
 	ITEM_OCCULT_NECKLACE(OCCULT_NECKLACE, OCCULT_NECKLACE_OR),
 	ITEM_OCCULT_ORNAMENT_KIT(OCCULT_ORNAMENT_KIT, OCCULT_NECKLACE_OR),
-	ITE_AMULET_OF_FURY(AMULET_OF_FURY, AMULET_OF_FURY_OR),
-	ITE_FURY_ORNAMENT_KIT(FURY_ORNAMENT_KIT, AMULET_OF_FURY_OR),
+	ITEM_AMULET_OF_FURY(AMULET_OF_FURY, AMULET_OF_FURY_OR),
+	ITEM_FURY_ORNAMENT_KIT(FURY_ORNAMENT_KIT, AMULET_OF_FURY_OR),
```
Comment (ThatGamerBlue) [inline runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java]: This'll likely get fixed upstream, we should let them deal with it.

File: runelite-api/src/main/java/net/runelite/api/AnimationID.java (hunk 1 above)
Comment (Kyleeld) [reply runelite-api/src/main/java/net/runelite/api/AnimationID.java]: removed + added comment to block_unarmed

File: runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java (hunk 2 above)
Comment (Kyleeld) [reply runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java]: reverted

**Pull request merged**